code/src/App.js

```diff
@@ -1,9 +1,41 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import { HappyThought } from './components/HappyThought';
+import { HappyForm } from './components/HappyForm';
+
+const url = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
```

Comment:
I have the same problem on my project about using '' and "". So you can also change those to make your code more clean and consistence.(: